taskfile.yaml

```diff
@@ -110,7 +110,7 @@ tasks:
       - task: push-image-helper
         vars:
           SOURCE: 'localhost/network-resources-injector:dev'
-          IMAGE: '{{.REGISTRY}}:5000/network-resources-injector:dev'
+          IMAGE: '{{.REGISTRY}}:5000/network-resources-injector-image:dev'
```

Comment:
This was a bug? Why is it different from the other images?

Reply:
Good catch, but we should fix it so that it's uniform at this level. I'll push a PR for the fix